### PR TITLE
README: Update outdated text and link

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,13 +134,20 @@ the target device. This includes both 64- and 32-bit toolchains. For the exact
 toolchain in use, please have a look at [toolchain.mk](https://github.com/OP-TEE/build/blob/master/toolchain.mk)
 and then look at the targets makefile (see [build.git](https://github.com/OP-TEE/build))
 to find out where the respective toolchain will be used. For example in the
-[QEMU makefile](https://github.com/OP-TEE/build/blob/master/qemu.mk#L12-L15) you
-will see:
+[QEMU makefile](https://github.com/OP-TEE/build/blob/master/qemu.mk#L6-L8) and
```
Comment (jforissier) [inline README.md]: L6-L9

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: README.md (hunk 1 above)
Comment (vchong) [inline README.md]: Oops. Will fix.

Comment (vchong) [inline README.md]: Thanks for catching!

**Comment by vchong** [comment]
Comment addressed and tags added.
Thanks!

**Pull request merged**